platform/server/src/main/webapp/static/js/bootstrap-page.js

```diff
@@ -234,5 +234,22 @@ function retrieveStatus() {
 }
 
 function submitForm() {
-    blockUI();
+    var dialog = new BootstrapDialog({
+        message: function(dialogRef){
+            var $message = $(
+                '<div class="splash-logo"><img src="./../../static/common/img/motech-logo.gif" alt="motech-logo"></div>' +
+                '<div class="clearfix"></div>' +
+                '<div class="splash-loader"><img src="./../../static/common/img/loadingbar.gif" alt="Loading..."></div>' +
+                '<div class="clearfix"></div>' + '<br>');
+                return $message;
```

Comment:
Lets get this HTML markup into a an actual HTML file somewhere, and out of the javascript